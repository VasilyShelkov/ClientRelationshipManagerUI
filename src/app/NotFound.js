import React from 'react';
import { Link } from 'react-router';

export default () => (
  <div>
    <h1>Oops! You&apos;ve ended up in a dark place</h1>
    <br />
    <Link to="/">Click here to return to normality</Link>
  </div>
 );
