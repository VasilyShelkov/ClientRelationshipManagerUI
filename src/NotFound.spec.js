import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter } from 'react-router';
import NotFound from './NotFound';

describe('src/NotFound.js', () => {
  it('renders the Not Found page', () => {
    expect(renderer.create(
      <BrowserRouter>
        <NotFound />
      </BrowserRouter>
    ).toJSON())
      .toMatchSnapshot();
  });
});
