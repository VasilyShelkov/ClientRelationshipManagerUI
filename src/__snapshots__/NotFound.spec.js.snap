exports[`src/NotFound.js renders the Not Found page 1`] = `
<div>
  <h1>
    Oops! You\'ve ended up in a dark place
  </h1>
  <br />
  <a
    className=""
    href="/"
    onClick={[Function]}
    style={Object {}}>
    Click here to return to normality
  </a>
</div>
`;
